### PR TITLE
Add Request Access to Admins functionality to non-accessible projects

### PR DESCRIPTION
**Pull request by mgs95**

### CODE REVIEW

File: app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styles from './AdminEmail.module.scss';
+import CopyToClipboard from '../../../../Components/CopyToClipboard/CopyToClipboard';
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx:2]: can we use the absolute path here?
```suggestion
import CopyToClipboard from 'Components/CopyToClipboard/CopyToClipboard';
```

File: app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  function onSendEmail() {
+    window.open(`mailto:${email}`);
+  }
+
+  return (
+    <div className={styles.adminEmail}>
+      <span>{email}</span>
+      <div className={styles.actions}>
+        <CopyToClipboard>{email}</CopyToClipboard>
+        <Button Icon={IconSend} label={''} onClick={() => onSendEmail()} />
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx:20]: we can omit the arrow function in the onClick prop, like this:
```suggestion
        <Button Icon={IconSend} label={''} onClick={onSendEmail} />
```

File: app/ui/src/Pages/Projects/components/Project/Project.tsx [hunk 3]
```diff
@@ -11,43 +11,68 @@ import { Link } from 'react-router-dom';
...
 
-const UpperBg: FC<Props> = ({ project }) => (
-  <div className={styles.sup}>
-    <div className={styles.bg}>
-      <div className={styles.bgBand} />
-    </div>
-    <div className={styles.content}>
-      <p className={styles.name}>{project.name}</p>
+function UpperBg({ project, showAdmins }: Props) {
+  function onContactAdmins() {
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/components/Project/Project.tsx:47]: I really don't know if it is a best practise in React but this function should be named: `handleContactAdmins`:
```suggestion
  function handleContactAdmins() {
```

File: app/ui/src/Pages/Projects/components/Project/Project.tsx [hunk 4]
```diff
@@ -11,43 +11,68 @@ import { Link } from 'react-router-dom';
...
+
+  return (
+    <div className={styles.sup}>
+      <div className={styles.bg}>
+        <div className={styles.bgBand} />
+      </div>
+      <div className={styles.content}>
+        <p className={styles.name}>{project.name}</p>
+        {project.needAccess && (
+          <div className={styles.contactInfo} onClick={() => onContactAdmins()}>
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/components/Project/Project.tsx:66]: we can omit (if we want) the arrow function:
```suggestion
          <div className={styles.contactInfo} onClick={handleContactAdmins}>
```

**Review by marco-gattelli — APPROVED** [review]

File: app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import CopyToClipboard from '../../../../Components/CopyToClipboard/CopyToClipboard';
+import { Button } from 'kwc';
+import IconSend from '@material-ui/icons/Send';
+import React from 'react';
+
+type Props = {
+  email: string;
+};
+function AdminEmail({ email }: Props) {
+  function onSendEmail() {
```
Comment (marco-gattelli) [inline app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx:11]: this also should be named: `handleSendEmail`:
```suggestion
  function handleSendEmail() {
```

File: app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx (hunk 1 above)
Comment (mgs95) [reply app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx]: Done

File: app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx (hunk 2 above)
Comment (mgs95) [reply app/ui/src/Pages/Projects/components/AdminEmail/AdminEmail.tsx]: Done

File: app/ui/src/Pages/Projects/components/Project/Project.tsx (hunk 4 above)
Comment (mgs95) [reply app/ui/src/Pages/Projects/components/Project/Project.tsx]: Done

**Pull request merged**